docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+:::{code} bash
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --time=4:00:00
+#SBATCH --job-name=MyJobName
+#SBATCH --mem=100G
+# SBATCH --partition=short
+
+# <commands to execute>
+:::
+
+**Job request: one node with exclusive use of a node**
+If you need exclusive use of a node, such as when you have a job that has high I/O requirements, you can use the exclusive flag. The example script below specifies the exclusive use of one node in the short partition for four hours:
+
+:::{code} bash
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --time=4:00:00
+#SBATCH --job-name=MyJobName
+#SBATCH --exclusive
+# SBATCH --partition=short
+
+# <commands to execute>
+:::
+
+(using-srun)=
+## Interactive Jobs: `srun` Command
+The `srun` command is used to submit an interactive job which runs a single task directly via the shell. This method is useful when you want to test a short computation or run an interactive session like a shell, Python, or an R terminal.
+
+### Syntax: `srun`
+
+:::{code} bash
+srun [options] [command]
+:::
+
+### Options and Usage: `srun`
+- **`n, --ntasks=<number>`**: specify the number of tasks
+- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
+- **`J, --job-name=<jobname>`**: specify a name for the job
+
+### Code Example for Interactive Job Submission
+:::{code} bash
+srun -N 1 -n 1 --pty bash
+:::
+
+This command starts an interactive bash shell on one node with one task.
+
+### Viewing Cluster Information
+:::{code} bash
+sinfo <options>
+:::
+
+This command allows to view partition and node information. Use option -a to view all partitions.
+
+:::{code} bash
+smap <options>
+:::
+
+This command allows to view details about the cluster in a visual format
+
+# Examples using srun
+
+The user needs to review the {ref}`hardware-overview` and [Partitions](https://northeastern-university-rc-public-documentation--33.com.readthedocs.build/en/33/hardware/partitions.html#partition-names) sections to be familiar with the available hardware and partition limits on Discovery. This way, user can tailor the request to fit both the needs of the job and the limits of partitions. For example, if the user specifies `--partition=short` and `--time=01:00:00`, it will result in an error because the time specified exceeds the limit for that partition.
+
+This simple `srun` example is to move to a *compute* node after you first log into the HPC:
+
+:::{code} bash
+srun --pty /bin/bash
+:::
+
+To request one node and one task for 30 minutes with X11 forwarding on the short partition, type:
+
+:::{code} bash
+srun --partition=short --export=ALL --nodes=1 --ntasks=1 --x11 --mem=10G --time=00:30:00 --pty /bin/bash
```

Comment:
Do we want to describe what the --export flag is doing? 

Also @manasvita can confirm if we want to encourage its use or not?

Reply:
Considering `ALL` is the default value for export anyhow, I just removed it for the sake of simplicity.